pkg/openstack/loadbalancer_sg.go

```diff
@@ -86,11 +86,8 @@ func applyNodeSecurityGroupIDForLB(network *gophercloud.ServiceClient, svcConf *
 			}
 
 			// Add the SG to the port
-			// TODO(dulek): This isn't an atomic operation. In order to protect from lost update issues we should use
-			//              `revision_number` handling to make sure our update to `security_groups` field wasn't preceded
-			//              by a different one. Same applies to a removal of the SG.
 			newSGs := append(port.SecurityGroups, sg)
-			updateOpts := neutronports.UpdateOpts{SecurityGroups: &newSGs}
```

Comment:
What's the intended retry logic here? I _guess_ relying on controller backoff is fine, maybe just worth a different error message, but for reference: https://github.com/openshift/cloud-network-config-controller/blob/master/pkg/cloudprovider/openstack.go#L765-L809